docs/dashboard.md

```diff
@@ -1,54 +1,207 @@
+    honorLabels: false
+    relabelings:
+    - sourceLabels: [__meta_kubernetes_endpoints_name]
+      regex: hami-.*
       replacement: $1
       action: keep
-    - source_labels: [__meta_kubernetes_pod_node_name]
+    - sourceLabels: [__meta_kubernetes_pod_node_name]
       regex: (.*)
-      target_label: node_name
+      targetLabel: node_name
       replacement: ${1}
       action: replace
-    - source_labels: [__meta_kubernetes_pod_host_ip]
+    - sourceLabels: [__meta_kubernetes_pod_host_ip]
       regex: (.*)
-      target_label: ip
+      targetLabel: ip
       replacement: $1
       action: replace
-- job_name: 'kubernetes-dcgm-exporter'
-    kubernetes_sd_configs:
-    - role: endpoints
-    relabel_configs:
-    - source_labels: [__meta_kubernetes_endpoints_name]
-      regex: dcgm-exporter
+
+#apply the file hami-device-plugin-svc-monitor.yaml
+root@controller01:~# kubectl apply -f hami-device-plugin-svc-monitor.yaml
+```
+
+```shell
+#Create the file hami-scheduler-svc-monitor.yaml
+root@controller01:~# touch hami-device-plugin-svc-monitor.yaml
+#The content of the file hami-scheduler-svc-monitor.yaml
+root@controller01:~# cat hami-scheduler-svc-monitor.yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: hami-scheduler-svc-monitor
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: hami-scheduler
+  namespaceSelector:
+    matchNames:
+      - "kube-system"
+  endpoints:
+  - path: /metrics
+    port: monitor
+    interval: "15s"
+    honorLabels: false
+    relabelings:
+    - sourceLabels: [__meta_kubernetes_endpoints_name]
+      regex: hami-.*
       replacement: $1
       action: keep
-    - source_labels: [__meta_kubernetes_pod_node_name]
+    - sourceLabels: [__meta_kubernetes_pod_node_name]
       regex: (.*)
-      target_label: node_name
+      targetLabel: node_name
       replacement: ${1}
       action: replace
-    - source_labels: [__meta_kubernetes_pod_host_ip]
+    - sourceLabels: [__meta_kubernetes_pod_host_ip]
       regex: (.*)
-      target_label: ip
+      targetLabel: ip
       replacement: $1
       action: replace
+
+#apply the file hami-scheduler-svc-monitor.yaml
+root@controller01:~# kubectl apply -f hami-scheduler-svc-monitor.yaml
+```
+
+```shell
+#Check the servicemonitors
+root@controller01:~# kubectl -n kube-system get servicemonitor
+NAME                             AGE
+hami-device-plugin-svc-monitor   28h
+hami-scheduler-svc-monitor       28h
 ```
 
-- reload promethues：
+# Confirm the final monitoring effect
+
+```shell
+#Create the file gpu-pod.yaml
+root@controller01:~# touch gpu-pod.yaml
+root@controller01:~# cat gpu-pod.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: gpu-pod-01
+spec:
+  restartPolicy: Never
+  containers:
+  - name: cuda-container
+    image: nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda10.2
+    resources:
+      limits:
+        nvidia.com/vgpu: 2 # requesting 2 vGPUs

```

Comment:
Should "nvidia.com/vgpu" be "nvidia.com/gpu"?

Reply:
I forgot to explain it, it depends on our own case. 
In order to distinguish from “nvidia.com/gpu” in nvidia-device-plugin, I used resourceName parameter and setted it's value to "nvidia.com/vgpu", such as: helm install hami hami-charts/hami --set resourceName=nvidia.com/vgpu  --set scheduler.kubeScheduler.imageTag=v1.23.10 -n kube-system

Reply:
> Should "nvidia.com/vgpu" be "nvidia.com/gpu"?

I forgot to explain it, it depends on our own case.
In order to distinguish from “nvidia.com/gpu” in nvidia-device-plugin, I used resourceName parameter and setted it's value to "nvidia.com/vgpu", such as: helm install hami hami-charts/hami --set resourceName=nvidia.com/vgpu --set scheduler.kubeScheduler.imageTag=v1.23.10 -n kube-system